src/SAWScript/CrucibleBuiltins.hs

```diff
@@ -170,12 +170,15 @@ crucible_llvm_verify bic opts lm nm lemmas checkSat setup tactic =
      let sym = cc^.ccBackend
      let nm' = fromString nm
      let llmod = cc^.ccLLVMModule
+     let stripSuffix (L.Symbol s) = L.Symbol (takeWhile (/= '.') s)
```

Comment:
Perhaps this should go from the end of the name? I'm wondering if one could write `f.foo.bar` and cause this to trip up.

Reply:
Hmm. Good question. Coming from C, it's not an issue, since `.` isn't a valid character in identifiers, and there's always at most one numeric suffix. Maybe other languages would use `.` for something, though?

Reply:
As crazy as it sounds, one could even use SAW to verify hand-written LLVM, in which case the identifier could be anything you like :stuck_out_tongue: I think going backwards from the end of the name is the more cautious option. I don't think it's a show-stopper though.

It sucks that so much metadata is stored in the flat, small, unstructured datatype that is a function's linkage name.

Reply:
Indeed.  It certainly is the case that LLVM intrinsics themselves are very happy to use the `.` character in their names.

Reply:
Yeah, going backwards does make more sense when considering LLVM that doesn't come from C. I'll change it.